### DISCUSSION TITLE
Inconsistent font in terminal

### DISCUSSION THREAD

**Discussion by mihaiginta** [starship/starship#4536] [Q&A]
Hello,
do you have any idea what I have to configure in order to get a consistent font when using starship? At the moment it looks like this: 
![image](https://user-images.githubusercontent.com/15912888/198227833-271598eb-cd70-4916-a126-068589f3a709.png).
These are my terminal settings:
![image](https://user-images.githubusercontent.com/15912888/198228389-2cfb5913-5f2c-494c-bbd9-7dcf1dadc866.png)

Thanks a lot

**Comment by jankatins** [comment]
You mean all non-bold?

**Reply by mihaiginta** [reply]
Yes

**Comment by chipbuster** [comment]
This is primarily a terminal emulator issue (as you can see in the screenshot, the output of `ls` is different with bold fonts as well, and starship has nothing to do with that).

I suspect the answer for your current terminal emulator is that you can't do that. Some terminal emulators will let you choose different font sets for bold/italic font. For those, you can sometimes set the bold fonts to be the same as the regular weight ones. I'm not aware of any way to do this for GNOME Terminal.

**Comment by davidkna** [comment]
I don't have a system with Gnome Terminal handy right now, but when installing the Fira Code nerd font, did you install all font-weight variants that came with it? It's possible that Gnome Terminal can't find the bold variant on your system.

**Reply by mihaiginta** [reply]
I will check in the next days this in the next days
